src/main/User/Services/GetUserDefaultIdService.java

```diff
@@ -0,0 +1,46 @@
+package User.Services;
+
+import Config.Message;
+import Config.Service;
+import Database.User.UserDao;
+import User.User;
+import User.UserMessage;
+import lombok.extern.slf4j.Slf4j;
+import java.util.Optional;
+
+@Slf4j
+public class GetUserDefaultIdService implements Service{
+    private final UserDao userDao;
+    private final String username;
+    private final String documentType;
```

Comment:
I would rather have this DocumentType be an enum value with a string representation